src/github.com/matrix-org/dendrite/common/config/appservice.go

```diff
@@ -188,6 +188,11 @@ func checkErrors(config *Dendrite) (err error) {
 			}
 		}
 
+		// Check if the url has trailing /'s. If so, remove them
+		for strings.HasSuffix(appservice.URL, "/") {
+			appservice.URL = strings.TrimSuffix(appservice.URL, "/")
+		}
```

Comment:
You don't actually need to have the guard there

Reply:
This covers the use case of multiple trailing `/`s though?

Reply:
Oh, it's a `for` loop. I need to learn to read. Anyway, you can just use `TrimRight`, which will strip off all trailing slashes.

Reply:
Oh, that's handy.